### PR TITLE
Users/shrverm/update legacy tasks m170

### PR DESCRIPTION
**Pull request by shadargee1982**
**Task name**: NpmV0 && NuGetInstallerV0

**Description**: Removed dependency on vsts-task-lib, which was causing warnings in the build pipelines such as "Warning: Use Cipheriv for counter mode of aes-256-ct". Switching to use azure-pipelines-task-lib fixes this.

**Documentation changes required:** N

**Added unit tests:** N

**Attached related issue:** Y https://dev.azure.com/mseng/AzureDevOps/_workitems/edit/1670738

**Checklist**:
- [ ] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/NpmV0/task.loc.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 0,
-    "Minor": 151,
+    "Minor": 172,
```
Comment (satbai) [inline Tasks/NpmV0/task.loc.json:12]: Change to 170 here as well

File: Tasks/NuGetInstallerV0/package.json [hunk 2]
```diff
@@ -19,7 +19,6 @@
   "dependencies": {
     "packaging-common": "file:../../_build/Tasks/Common/packaging-common-1.0.1.tgz",
     "utility-common": "file:../../_build/Tasks/Common/utility-common-1.0.2.tgz",
-    "vsts-task-lib": "2.6.0",
```
Comment (satbai) [inline Tasks/NuGetInstallerV0/package.json:22]: do we need to add 'azure-pipelines-task-lib' here?

File: Tasks/NuGetInstallerV0/task.loc.json [hunk 3]
```diff
@@ -3,12 +3,13 @@
...
   "friendlyName": "ms-resource:loc.friendlyName",
   "description": "ms-resource:loc.description",
+  "helpUrl": "https://aka.ms/NuGetTask",
   "helpMarkDown": "ms-resource:loc.helpMarkDown",
   "category": "Package",
   "author": "Microsoft Corporation",
   "version": {
     "Major": 0,
-    "Minor": 146,
+    "Minor": 172,
```
Comment (satbai) [inline Tasks/NuGetInstallerV0/task.loc.json:12]: same here as above

File: Tasks/NuGetInstallerV0/package.json (hunk 2 above)
Comment (shadargee1982) [reply Tasks/NuGetInstallerV0/package.json:22]: no, it gets pulled in with utility-common.

**Pull request merged**